database-plugins/src/main/java/io/cdap/plugin/DBUtils.java

```diff
@@ -232,13 +247,21 @@ private static Schema getSchema(String typeName, int sqlType, int precision, int
         break;
 
       case Types.NUMERIC:
```

Comment:
What is the difference between numeric and decimal type? Does this change need to be applied to numeric too?